CourseGrab/ViewControllers/Search/SearchDetailTableViewCell.swift

```diff
@@ -17,24 +17,46 @@ class SearchDetailTableViewCell: UITableViewCell {
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
 
-        statusView.backgroundColor = .red
-        statusView.layer.cornerRadius = 2
+        selectionStyle = .none
+        setupStatusView(isOpen: false)
+        setupTitleLabel()
+        setupTrackingButton()
+    }
+
+    private func setupStatusView(isOpen: Bool) {
+        if isOpen {
+            statusView.backgroundColor = .courseGrabGreen
+            statusView.layer.cornerRadius = 8
+        } else {
+            statusView.backgroundColor = .courseGrabRuby
+            statusView.layer.cornerRadius = 2
+        }
         contentView.addSubview(statusView)
+
         statusView.snp.makeConstraints { make in
             make.height.width.equalTo(16)
             make.left.equalToSuperview().offset(20)
             make.centerY.equalToSuperview()
         }
+    }
 
+    private func setupTitleLabel() {
+        titleLabel.font = ._14Semibold
         contentView.addSubview(titleLabel)
+
         titleLabel.snp.makeConstraints { make in
             make.left.equalTo(statusView.snp.right).offset(12)
             make.centerY.equalToSuperview()
         }
+    }
 
+    private func setupTrackingButton() {
         trackingButton.layer.cornerRadius = 2
-        trackingButton.backgroundColor = .blue
+        trackingButton.layer.borderWidth = 1
+        trackingButton.layer.borderColor = UIColor.courseGrabRuby.cgColor
+        trackingButton.titleLabel?.font = ._12Semibold
         contentView.addSubview(trackingButton)
+        
         trackingButton.snp.makeConstraints { make in
             make.height.equalTo(24)
```

Comment:
We should move all constants to the top of the file and add 
``` 
// MARK: - Constants 
let trackingButtonHeight: CGFloat = 24
```

Reply:
i've actually changed my stance on this recently. i think we should only make them constants if we use the same value more than once and if we do make it a constant, to just put it at the top of the function instead of placing it at the top of the class

Reply:
main reasoning is that if the values are just used once, making them into constants is just extra work with no real benefit

Reply:
Sounds good!